src/boot.mjs

```diff
@@ -3,15 +3,33 @@ import "dotenv/config";
 import { Worker } from "worker_threads";
 import { resolve } from "path";
 import { env } from "process";
+import { config as configSchema } from "@neume-network/schema";
+import Ajv from "ajv";
+import addFormats from "ajv-formats";
 
 import { __dirname } from "./node_filler.mjs";
+import logger from "./logger.mjs";
 import * as strategies from "./strategies/src/index.mjs";
 import * as environment from "./environment.mjs";
 import * as disc from "./disc.mjs";
 
 const workerPath = resolve(__dirname, "./worker_start.mjs");
+const log = logger("boot");
 
-export async function createWorker() {
+async function getConfig(configFile) {
```

Comment:
pls test this function for happy and unhappy paths thx

Reply:
I thought about adding tests for this function. I was confused so skipped it. I was afraid that adding too many tests may make future changes difficult. I don't want even minor refactors to break tests. I will add tests for this one though.

Reply:
> I was afraid that adding too many tests may make future changes difficult. I don't want even minor refactors to break tests.

I understand the anxiety around cementing the code base in tests. But I see it mostly as something that makes sense as an argument but isn't true in practice. Good tests allow broad refactors as you get higher confidence over knowing what works and what doesn't.